.env.local.example

```
@@ -33,6 +33,10 @@ ONBOARDING_STATE = open
 # Enable Twelve market data (careful, this will use your API credits)
 TWELVE_DATA_API_KEY =
 
+# Optional: enables spot-price valuations for manual physical-gold accounts.
+# GoldAPI's free plan is capped; Sure caches one XAU quote per currency per day.
+GOLD_API_KEY =
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)